### ISSUE TITLE
Alternative Web Control Interfaces

### ISSUE THREAD

**Issue by Mr-DaveDev** [Motion-Project/motionplus#2]
The web control interface in MotionPlus is very basic.  As an enhancement, users can now specify their own web control page for MotionPlus to use.  Samples can be included so users can easily switch between them.  

The community is encouraged to create alternative web pages for MotionPlus to use and test them via the `webcontrol_html` and 'webcontrol_interface' option 3 parameter

**Comment by husain3** [comment]
Is there a reason why you wanted to create such a basic web control page? Would you be interested in having a webpage that was a little easier to use?

**Comment by Mr-DaveDev** [comment]
Yes, that was the idea of having the page as a parameter.  The basic nature of the plus interface is still much further beyond the regular Motion code interface.  One consideration is that once MotionPlus is all set up, it is recommended that that the parameters be disabled and then you'd be able to put on the web without concern.

**Comment by husain3** [comment]
Are there any plans regarding video playback via web interface? I know for me I have to scp all the necessary .mkv files from the motion record dir lol. I wanted to make a quick web control interface with react because I find all the different parameters and configurations hard to understand.

**Comment by Mr-DaveDev** [comment]
You can do a pull request illustrating the changes and your suggested version of the page and code changes to support video playback.

**Comment by extorn** [comment]
I upload all the recorded images to a separate server using WEBDAV. My upload script is triggered by the helpful "on_movie_end" event in my cameraX.conf file
